app/(golems)/golems/page.tsx

```
@@ -0,0 +1,506 @@
+    '34': '#6ab0f3',
+    '36': '#40d4d4',
+  };
+  const regex = /\x1b\[(\d+)m/g;
+  let lastIndex = 0;
+  let currentColor = '';
+  let match;
+
+  while ((match = regex.exec(raw)) !== null) {
+    if (match.index > lastIndex) {
+      const text = raw.slice(lastIndex, match.index);
+      parts.push(currentColor ? <span key={key++} style={{ color: currentColor }}>{text}</span> : <span key={key++}>{text}</span>);
```

Biome (2.3.14)

[error] Unexpected control character in a regular expression.

Control characters are unusual and potentially incorrect inputs, so they are disallowed.

(lint/suspicious/noControlCharactersInRegex)